### PR TITLE
Better handling of windows with .nws file.

### PR DESCRIPTION
**Pull request by willrogers**
On the first run, if a .nws file was launched, reuse a workbench window.
Otherwise, open a new one.

@mfurseman @nickbattam anyone want to merge with their eyes shut?

### CODE REVIEW

**Comment by nickbattam** [comment]
If your initial launch has no arguments `css.sh` will run through the "default is databrowser.nws" file path and open with a nws file but NOT set the nws_file flag. Is this intentional?

File: features/org.csstudio.dls.product.feature/rootfiles/css.sh [hunk 1]
```diff
@@ -137,7 +141,7 @@ fi
 
 # OPI file and related options.
 if [[ -n $macros ]] || [[ -n $links ]]; then
-    if [[ -z $opifile ]]; then
+    if [[ -z $runfile ]]; then
         echo "Macros and links arguments require an opi file argument."
```
Comment (nickbattam) [inline features/org.csstudio.dls.product.feature/rootfiles/css.sh]: Message needs updating: .nws, .plt or .opi file is required.

**Comment by nickbattam** [comment]
What happens if both `-l` links and an `.nws` file are specified? Does it add the explicit links to those defined in the .nws file? (slightly off topic)

**Comment by willrogers** [comment]
No it's not intentional - good spot.

**Comment by willrogers** [comment]
The `-l` argument would still be honoured //on first launch//, independently of the contents of the `.nws` file.

**Pull request merged**